frontend/docsettings.lua

```diff
@@ -350,7 +404,8 @@ end
 
 --- Removes empty sidecar dir.
 function DocSettings:removeSidecarDir(doc_path, sidecar_dir)
-    if sidecar_dir == self:getSidecarDir(doc_path, "doc") then
+    if sidecar_dir == self:getSidecarDir(doc_path, "doc") or
+        sidecar_dir == self:getSidecarDir(doc_path, "hash") then
```

Comment:
No, this must be with "dir". So you can just remove `or` condition here.

But must be added to `purge()`.